ortools/linear_solver/linear_solver.cc

```diff
@@ -1511,6 +1511,32 @@ MPConstraint* MPSolver::MakeRowConstraint(const LinearRange& range,
   return constraint;
 }
 
+MPConstraint* MPSolver::MakeIndicatorConstraint(
+    double lb, double ub, const std::string& name,
+    const MPVariable* indicator_variable, bool indicator_value) {
+  if (!indicator_variable->integer() || indicator_variable->lb() != 0 ||
```

Comment:
To be consistent with e.g `MPObjective::SetCoefficient` and `MPConstraint::SetCoefficient`, you'd need the following lines

Although the 2nd line is currently included in the 1st.

```cpp
DLOG_IF(DFATAL, !interface_->solver_->OwnsVariable(indicator_variable)) << indicator_variable;
if (indicator_variable == nullptr) return nullptr;
```

Reply:
good catch!